tests/test_commands.py

```diff
@@ -234,6 +233,62 @@ def test_nickserv_and_memoserv(alice, bob, wait_until):
     wait_until(lambda: "send Bob hello there\n" in bob.text())
 
 
+@pytest.mark.skipif(
+    os.environ["IRC_SERVER"] == "hircd",
+    reason="hircd doesn't support away notifications",
+)
+def test_away_status(alice, bob, wait_until):
+    switch_to_channel_view(bob, "#autojoin")
```

Comment:
This does nothing. The test is showing the `#autojoin` channel by default. (You can check this by inserting `import pdb; pdb.set_trace()` to the beginning of the test). There's another unnecessary `switch_to_channel_view()` call below.